src/SB/Game/zNPCFXCinematic.h

```diff
@@ -190,6 +245,9 @@ struct NPCCone
     F32 uv_tip[2];
     F32 uv_slice[2];
     void TextureSet(RwRaster* raster);
+    void UVSliceSet(F32, F32);
+    void UVBaseSet(F32, F32);
+    void ColorSet(RwRGBA, RwRGBA);
```

Comment:
Please update these headers with identifiers for each function parameter so that it matches the implementation identifiers you used when you wrote the function in the CPP file.